bower.json

```diff
@@ -27,23 +27,22 @@
     "url": "git://github.com/openshift/origin-web-common.git"
   },
   "dependencies": {
-    "angular": "1.5.11 < 2.0.0",
-    "angular-sanitize": "1.5.11 < 2.0.0",
-    "angular-utf8-base64": "0.0.5",
-    "hawtio-core": "2.0.11",
-    "hawtio-extension-service": "2.0.2",
-    "jquery": "2.1.4",
-    "kubernetes-container-terminal": "1.0.2",
-    "lodash": "3.10.1",
-    "messenger": "1.4.1",
-    "microplugin": "0.0.3",
-    "openshift-object-describer": "1.1.2",
-    "sifter": "0.4.1",
-    "uri.js": "1.18.0"
+    "angular": "~1.5.11",
+    "angular-sanitize": "~1.5.11",
+    "angular-utf8-base64": "~0.0.5",
+    "hawtio-core": "~2.0.37",
+    "hawtio-extension-service": "~2.0.2",
+    "jquery": "~2.1.4",
+    "kubernetes-container-terminal": "~1.0.3",
```

Comment:
I don't think you need this dependency either... We might review these more closely to see if there are others.